packages/components/src/components/tag/tag.tsx

```diff
@@ -18,9 +18,9 @@ import classNames from 'classnames';
 })
 export class Tag {
   /** (optional) Tag size */
-  @Prop() size?: string = '';
+  @Prop() size?: '' | 'small' = '';
```

Comment:
do we really need to define an empty string here as a possible value? the `?` makes the prop optional already. I think it would make sense to change this to `@Prop() size?: 'small';` What do you think?

Reply:
Actually no we won't need this. That's a good addition.
Just checked if the classNames function would work correctly with a possible `undefined` state of size but it does.